lang/c/c.talon

```diff
@@ -135,17 +48,17 @@ push brackets:
     key(enter)
     key(enter)
     edit.up()
-
-# Declare variables or structs etc.
-# Ex. * int myList
+    
+    # Declare variables or structs etc.
+    # Ex. * int myList
```

Comment:
not really that important  for this review i guess, but i've noticed a few similar places where suddenly the comments are indented like this. i didn't flag them, but suddenly curious if it's a side effect of the migration script or something?

Reply:
yes this is probably a bug in the migration script

Reply:
I'm pretty indifferent here fwiw. I'd be tempted to call the migration script Good Enough(tm) as-is

Reply:
I've cleaned these up. There weren't many.